talk/owt/sdk/base/peerconnectionchannel.cc

```diff
@@ -216,7 +216,38 @@ void PeerConnectionChannel::OnMessage(rtc::Message* msg) {
       rtc::ScopedRefMessageData<MediaStreamInterface>* param =
           static_cast<rtc::ScopedRefMessageData<MediaStreamInterface>*>(
               msg->pdata);
-      peer_connection_->RemoveStream(param->data());
+      MediaStreamInterface* stream = param->data();
+      for (const auto track : stream->GetAudioTracks()) {
```

Comment:
Prefer to add a new function for getting sender, which will avoid duplicated code.